linkcheck/HtmlParser/htmlparse.c

```diff
@@ -2460,35 +2460,36 @@ MOD_INIT(htmlsax) {
         /* init error */
         PyErr_Print();
     }
-    if ((m = PyImport_ImportModule("linkcheck.HtmlParser")) == NULL) {
+    PyObject* h = NULL;
+    if ((h = PyImport_ImportModule("linkcheck.HtmlParser")) == NULL) {
```

Comment:
What is the reason for renaming `m` to `h` in the rest of this function?

Reply:
To do with the return value?:

https://github.com/linkchecker/linkchecker/blob/7c0d8603368b33c4c409a30434ba9f9e93b4d898/linkcheck/HtmlParser/htmlparse.c#L2494

Reply:
Thank you!

I see, on Python 2 MOD_SUCCESS_VAL(m) is defined to be empty, so we need to preserve the actual module we're defining and return it on Python 3 only.  Which explains why the old code was not buggy on Python 2, and why it needed to be changed to support Python 3.